### PR TITLE
Added a card and bug fix

### PR DESCRIPTION
**Pull request by aoaaceai**

### CODE REVIEW

File: src/main/java/eod/effect/Summon.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+    @Override
+    public Player getPlayer() {
+        return null;
+    }
+
+    @Override
+    public void teardown() {
+
```
Comment (secminhr) [inline src/main/java/eod/effect/Summon.java]: Make player and object null.

File: src/main/java/eod/warObject/other/concrete/Drone.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    @Override
+    public CanAttack getAttacker() {
+        return null;
+    }
+
+    @Override
+    public void attack() {
+        int plusAttack = WarObject(player.getBoard()).which(OwnedBy(player)).which(Being(Drone.class)).which(InRangeOf(this)).get().length*2;
+        attack += plusAttack;
```
Comment (secminhr) [inline src/main/java/eod/warObject/other/concrete/Drone.java]: Only add 2.

File: src/main/java/eod/warObject/other/concrete/Drone.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+    @Override
+    public int getAttackRange() {
+        return 1;
+    }
+
+    @Override
+    public void teardown() {
+        super.teardown();
+        status = null;
```
Comment (secminhr) [inline src/main/java/eod/warObject/other/concrete/Drone.java:98]: Clear status first.

**Comment by aoaaceai** [comment]
will be solved in the next pr. No idea why that wasn't added to this commit.
Probably because I've messed up the version control again lol

File: src/main/java/eod/effect/Summon.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    public Point from(ArrayList<Point> points) {
+        return on(askToSelectFrom(points));
+    }
+
+
+    @Override
+    public Player getPlayer() {
+        return null;
```
Comment (secminhr) [inline src/main/java/eod/effect/Summon.java:31]: Not sure if this is intended...

**Review by secminhr — APPROVED** [review]
Great

**Pull request merged**